assets/javascripts/admin_assets.js

```diff
@@ -30,6 +30,9 @@ function setupAdminAssets() {
                     $('#assets-status').text('failed to load');
                 }
             },
+	    createdRow: function (row, data) {
+	        $(row).attr("id", "asset_" + data.id);
```

Comment:
I would write it as `row.id = 'asset_' + data.id`. We don't need to use jQuery for every tiny thing.

Reply:
As your comment below, if we do not remove the row which the deleted asset belongs to, the code is not needed.